app/Base/configs/sentry.ts

```diff
@@ -5,40 +5,26 @@ import { Integrations } from '@sentry/tracing';
 import browserHistory from '#base/configs/history';
 import routes from '#base/configs/routes';
 import {
-    isBeta,
-    isAlpha,
-    isNightly,
     isDev,
+    appCommitHash,
+    appName,
+    sentryAppDsn,
+    deepEnvironment,
+    apiEndpoint,
 } from '#base/configs/env';
 
-const appCommitHash = process.env.REACT_APP_COMMITHASH;
-const appName = process.env.MY_APP;
-
-const sentryDsn = (() => {
-    const keyFromEnv = process.env.REACT_APP_SENTRY_DSN;
-    if (keyFromEnv) {
-        return keyFromEnv;
-    }
-    if (isBeta || isAlpha || isNightly) {
-        return 'https://9a60f35c6a1c45fe999727c5f6f7229c@sentry.io/1220157';
-    }
-    return undefined;
-})();
-
-const env = process.env.REACT_APP_DEEP_ENVIRONMENT;
-const sentryConfig: BrowserOptions | undefined = sentryDsn ? {
-    dsn: sentryDsn,
+const sentryConfig: BrowserOptions | undefined = sentryAppDsn ? {
+    dsn: sentryAppDsn,
     release: `${appName}@${appCommitHash}`,
-    environment: env,
+    environment: deepEnvironment,
     debug: isDev,
     // sendDefaultPii: true,
     normalizeDepth: 5,
     tracesSampleRate: 0.2,
     integrations: [
         new Integrations.BrowserTracing({
-            // NOTE: process.env.REACT_APP_API_END is actually the domain
-            // for the api endpoint
-            tracingOrigins: ['localhost', process.env.REACT_APP_API_END as string],
+            // NOTE: apiEndpoint is actually the domain for the api endpoint
+            tracingOrigins: ['localhost', apiEndpoint as string],
```

Comment:
Let's not cast this there, but on the env file.
This way we know what env variables are required